countdown_timer.jsx

```diff
@@ -126,7 +126,7 @@ var CountdownTimer = React.createClass({
     var timeRemaining = this.state.timeRemaining;
 
     return (
-      <div className='timer'>
+      <div className={this.props.setClass || 'timer'}>
```

Comment:
This should be ok, but looking at this I'm wondering if this.props.appendClassname should take either a string or an array, then we can append accordingly?